### PR TITLE
add jupyter notebook for confidence calibration model training

### PR DESCRIPTION
**Pull request by rongzhangibm**
This notebook shows how to train a confidence calibration model.

### CODE REVIEW

**Review by bhavani105 — APPROVED** [review]
LGTM

File: notebooks/confidence_calibration/confidence_calibration.ipynb [hunk 1]
```diff
@@ -0,0 +1,1875 @@
...
+    "train_batch_size = 8             # Set this to change the number of features per batch during training.\n",
+    "eval_batch_size = 8              # Set this to change the number of features per batch during evaluation.\n",
+    "gradient_accumulation_steps = 8  # Set this to effectively increase training batch size.\n",
+    "max_train_samples = 300          # Set this to use a subset of the training data (or None for all).\n",
+    "max_eval_samples = 50            # Set this to use a subset of the evaluation data (or None for all).\n",
+    "num_train_epochs = 1             # Set this to change the number of training epochs.\n",
+    "fp16 = False                     # Set this to true to enable fp16 (hardware support required).\n",
+    "\n",
+    "# Parameters for coonfidence model training (feel free to leave as default).\n",
+    "relative_confidence_train_size = 0.1          # Set this to change the relative size of confidence model train set split from original train set.\n",
```
Comment (seirasto) [inline notebooks/confidence_calibration/confidence_calibration.ipynb:72]:
```suggestion
    "relative_confidence_train_size = 0.1          # Set this to change the relative size of the confidence model train set split from the original train set.\n",
```

File: notebooks/confidence_calibration/confidence_calibration.ipynb [hunk 2]
```diff
@@ -0,0 +1,1875 @@
...
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "bb81a266",
+   "metadata": {},
+   "source": [
+    "## Printing Output\n",
+    "\n",
+    "Here we print out the answers with confidence scores. Please be noted that this notebook is only for the purpose of showing how to train a confidence calibration model. So we select a very small number of examples for both MRC and confidence model training. This leads to the result that most of the confidence scores are low. Users are welcome to modify this notebook to use a larger training set in order to obtain more reasonable confiidence scores. \n"
```
Comment (seirasto) [inline notebooks/confidence_calibration/confidence_calibration.ipynb:1724]: Some typos. I think you can show even less examples here - maybe 5-10?
```suggestion
    "Here we print out the answers with the confidence scores. Please note that this notebook is only for the purpose of showing how to train a confidence calibration model. So we select a very small number of examples for both MRC and confidence model training. This leads to the result that most of the confidence scores are low. Users are welcome to modify this notebook to use a larger training set in order to obtain more reasonable confidence scores. \n"
```

**Review by seirasto — APPROVED** [review]
LGTM. I've made a couple of typo suggestions.

**Review by mihaela-bornea — APPROVED** [review]
LGTM

**Pull request merged**